Gordon360/Program.cs

```diff
@@ -47,10 +48,15 @@
 builder.Services.AddScoped<IProfileService, ProfileService>();
 builder.Services.AddScoped<IAddressesService, AddressesService>();
 builder.Services.AddScoped<RecIM.IActivityService, RecIM.ActivityService>();
-
-
+builder.Services.AddScoped<RecIM.ISeriesService, RecIM.SeriesService>();
+builder.Services.AddScoped<RecIM.IMatchService, RecIM.MatchService>();
 
 builder.Services.AddMemoryCache();
+builder.Services.AddMvc().AddNewtonsoftJson(options =>
```

Comment:
What does this do and is it necessary?

Reply:
The reference loop was an issue derived from the very first iteration of the GET routes where I just returned Models. So now that we use ViewModels we won't need it! Good Catch :)